scripts/build-tag-push-ecr.sh

```diff
@@ -7,7 +7,7 @@ echo "building image(s)"
 cd "${WORKING_DIRECTORY}"
 echo "Packaging app in /$WORKING_DIRECTORY"
 
-docker build -t "$ECR_REGISTRY/$ECR_REPO_NAME:$GITHUB_SHA" .
+docker build -t "$ECR_REGISTRY/$ECR_REPO_NAME:$GITHUB_SHA" -f $DOCKERFILE .
```

Comment:
```suggestion
docker build -t "$ECR_REGISTRY/$ECR_REPO_NAME:$GITHUB_SHA" -f "$DOCKERFILE" .
```

Should quote this to be safe

Reply:
Cheers @CharlesIC